packages/config/scripts/checkVerifiedContracts/tasks.ts

```diff
@@ -13,67 +18,22 @@ export async function verifyContracts(
   logger: Logger,
 ): Promise<VerificationMap> {
   logger.info(`Processing ${addresses.length} addresses.`)
-  const result: VerificationMap = {}
 
-  const { taskQueue, queueStatus } = prepareTaskQueueWithQuickFail(
-    async (address: EthereumAddress) => {
+  const verificationPromises = addresses.map(
+    (address) => async (): Promise<[string, boolean]> => {
+      if (previouslyVerified.has(address) || manuallyVerified.has(address)) {
+        return [address.toString(), true]
+      }
+
       logger.info(`Checking ${address.toString()}...`)
       const isVerified = await isContractVerified(etherscanClient, address)
-      result[address.toString()] = isVerified
+      return [address.toString(), isVerified]
     },
-    logger,
-    workersCount,
   )
 
-  addresses.forEach((address) => {
-    if (previouslyVerified.has(address) || manuallyVerified.has(address)) {
-      result[address.toString()] = true
-    } else {
-      taskQueue.addToBack(address)
-    }
+  const verification = await promiseAllPlus(verificationPromises, logger, {
+    maxConcurrency: workersCount,
   })
 
-  await taskQueue.waitTilEmpty()
-  if (queueStatus.errorOccurred) {
-    throw new Error('An error occurred while fetching verification status.')
-  }
-  return result
-}
-
-interface QueueStatus {
-  errorOccurred: boolean
-}
-
-// Returns a queue that discards remaining tasks as soon as one fails.
-function prepareTaskQueueWithQuickFail(
-  task: (a: EthereumAddress) => Promise<void>,
-  logger: Logger,
-  workersCount: number,
-): {
-  taskQueue: TaskQueue<EthereumAddress>
-  queueStatus: QueueStatus
-} {
-  const queueStatus: QueueStatus = {
-    errorOccurred: false,
-  }
-
-  const taskQueue = new TaskQueue(
-    async (address: EthereumAddress) => {
-      if (!queueStatus.errorOccurred) {
-        await task(address)
-      } else {
-        logger.warn('Job skipped due to previous error.')
-      }
-    },
-    logger,
-    {
-      workers: workersCount,
-      shouldRetry: (_) => {
-        queueStatus.errorOccurred = true
-        return { retry: false }
-      },
-    },
-  )
-
-  return { taskQueue, queueStatus }
+  return Object.fromEntries(verification)
```

Comment:
My IDE screams that `verification` has `any` type. I'm not sure why our lint doesn't pick it up here.

Reply:
rebuild the project, perhaps. Works for me (TM)

Reply:
Restarting IDE helped :/